lib/typeform_data/requestor.rb

```diff
@@ -4,6 +4,8 @@
 module TypeformData
   module Requestor
 
+    RETRY_EXCEPTIONS = [Net::HTTPServiceUnavailable, Errno::ECONNREFUSED].freeze
```

Comment:
The PR was originally intended to address issues like transient `OpenSSL::SSL::Error`'s happening during our HTTP request. I just realized that my original implementation didn't actually address this problem (since the SSL errors didn't raise `TypeformData::UnexpectedError`). Do you think there's a good way to allow the API consumer to specify the exceptions to retry on?